pkg/vulnsrc/rootio/rootio_test.go

```diff
@@ -171,6 +172,7 @@ func TestVulnSrc_Get(t *testing.T) {
 			args: args{
 				osVer:   "11",
 				pkgName: "openssl",
+				arch:    "amd64",
```

Comment:
this change can confusing (debian advisories don't use arches).
So let's add arch only for rocky related advisories.